compiler/noirc_driver/src/lib.rs

```diff
@@ -290,20 +278,19 @@ pub fn add_dep(
 pub fn check_crate(
     context: &mut Context,
     crate_id: CrateId,
-    check_options: &CheckOptions,
+    options: &CompileOptions,
 ) -> CompilationResult<()> {
-    let options = &check_options.compile_options;
-
     let macros: &[&dyn MacroProcessor] =
         if options.disable_macros { &[] } else { &[&aztec_macros::AztecMacro] };
 
     let mut errors = vec![];
+    let error_on_unused_imports = true;
     let diagnostics = CrateDefMap::collect_defs(
         crate_id,
         context,
         options.debug_comptime_in_file.as_deref(),
         options.arithmetic_generics,
-        check_options.error_on_unused_imports,
+        error_on_unused_imports,
```

Comment:
left in from debugging the stdlib?

Reply:
Ah, no. It's that way so it only warns on the root crate and not on dependencies.